x-pack/plugins/security_solution/public/common/components/exceptions/viewer/exception_item_card/exception_item_card_meta.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import { EuiAvatar, EuiBetaBadge, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+import { ExceptionListItemSchema } from '@kbn/securitysolution-io-ts-list-types';
+
+import * as i18n from './translations';
+import { FormattedDate, FormattedRelativePreferenceDate } from '../../../formatted_date';
+
+export interface ExceptionItemCardMetaInfoProps {
+  item: ExceptionListItemSchema;
+  dataTestSubj: string;
+}
+
+export const ExceptionItemCardMetaInfo = memo<ExceptionItemCardMetaInfoProps>(
+  ({ item, dataTestSubj }) => {
+    return (
+      <EuiFlexGroup
+        alignItems="center"
+        responsive={false}
+        gutterSize="s"
+        data-test-subj={dataTestSubj}
+      >
+        <EuiFlexItem grow={false}>
+          <MetaInfoDetails
+            fieldName="created_by"
+            label={i18n.EXCEPTION_ITEM_CREATED_LABEL}
+            value1={
+              <FormattedDate fieldName="created_by" value={item.created_at} dateFormat="MM/DD/YY" />
```

Comment:
Should we specifically use this format here? As I understand it's [US only](https://iso.mit.edu/americanisms/date-format-in-the-united-states/#:~:text=The%20United%20States%20is%20one,yyyy%2Dmm%2Ddd).)

Also in Kibana Advanced settings, we have Date format - the user can specify how they want present dates, maybe it's better to use this one?

Reply:
That's a great point. I updated to not specify so that it should fall back to the users date preference.